src/update_parameters.jl

```diff
@@ -137,18 +147,15 @@ function update_parameter_in_quadratic_constraints_pc!(model::Optimizer)
             end
         end
         if param_constant != zero(Float64)
-            set = MOI.get(
```

Comment:
Ah. The old get-set problem with Gurobi :(

Reply:
Unfortunately this is a bottleneck in the previous implementation